app/views/videos/show/_actions.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="btn-module">
+  <% review_class = 'btn-block' %>
+  <% if translation.nil? %>
+    <%= link_to 'Assign to Me', video_translations_path(video), :class => 'btn btn-primary btn-left', :'data-method' => :post %>
+    <% review_class = 'btn-right' %>
+  <% elsif not translation.complete %>
+    <%= link_to 'Unassign Me', '#', :class => 'btn btn-danger btn-left', :data => { :toggle => 'modal', :target => '#deleteModal' } %>
+    <% review_class = 'btn-right' %>
+
+    <div class="modal fade" id="deleteModal" tabindex="-1" role="dialog" aria-labelledby="deleteModalLabel" aria-hidden="true">
+      <div class="modal-dialog">
+        <div class="modal-content">
```

Comment:
Could add module class to modal-content so that we have consistent rounded corners for larger entities.